scripts/run_molecule

```diff
@@ -45,14 +46,26 @@ for rolepath in ${ROLES_LIST}; do
     test -d ${role} || continue
     echo "Running molecule against: ${role}"
     pushd $role
-    case ${USE_VENV} in
-        y|yes|true)
-            ${HOME}/test-python/bin/molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
-            ;;
-        *)
-            molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
-            ;;
-    esac
+    MOLECULE_RESULTS=$(\
+        case ${USE_VENV} in
+                y|yes|true)
+                    ${HOME}/test-python/bin/molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
+                    ;;
+                *)
+                    molecule -c "${PROJECT_DIR}${MOLECULE_CONFIG}" ${TEST_VERBOSITY} test --all
+                    ;;
+        esac ) || true
+        if [[ $MOLECULE_RESULTS =~ failed=[1-9] ]]; then
+            MOLECULE_FAILURES=$MOLECULE_FAILURES"${role}\n"
```

Comment:
```suggestion
            MOLECULE_FAILURES="${MOLECULE_FAILURES}${role}\n"
```
Didn't test the specific `"` location change, but it seems more "bash". At least, the `{}` should be used for sure.